vaadin-map-flow-parent/vaadin-map-flow/src/main/resources/META-INF/resources/frontend/vaadin-map/synchronization/styles.js

```diff
@@ -0,0 +1,101 @@
+import Fill from "ol/style/Fill";
+import Stroke from "ol/style/Stroke";
+import Style from "ol/style/Style";
+import { Icon } from "ol/style";
+import {
+  convertToCoordinateArray,
+  convertToSizeArray,
+  createOptions,
+} from "./util";
+
+export function synchronizeFill(target, source, _context) {
+  if (!target) {
+    target = new Fill();
+  }
+
+  target.setColor(source.color);
+
+  return target;
+}
+
+export function synchronizeStroke(target, source, _context) {
+  if (!target) {
+    target = new Stroke();
+  }
+
+  target.setColor(source.color);
+  target.setWidth(source.width);
+
+  return target;
+}
+
+function synchronizeImageStyle(target, source, _context) {
```

Comment:
`synchronizeImageStyle` it looks like this is not added to the lookup table in the `index.js` file.

Reply:
That was intentional. The Image style is only a base class in OpenLayers, so we don't want it to be used directly in a configuration. Similarly the Java class pendant is declared as abstract, so it can not be instantiated. So the only use-case for `synchronizeImageStyle` is to be called from sync functions of sub-classes (see `synchronizeIcon`), which is why we don't need to expose it.